### ISSUE TITLE
Blogify does not support multi language websites ?

### ISSUE THREAD

**Issue by GHPaulB** [matfish2/craft-blogify#2]
Hi,

I have a multi language website on craft cms. 
Every language is hosted as a separate site.
It worked perfectly, until I installed Blogify.

Blogify installs its folders and files directly in the template folder, on the same level as the language sites.
It should do so inside each (or at least one) language site.
The plugin does not provide a possibility to choose the site in which I want it to be installed.

My multi language website is dead as long as I have Blogify installed, so, unfortunately, I had to uninstall it. 
Is there an easy fix ? Or is it money spent for nothing…

Kind regards,
Paul

<img width="422" alt="Schermafbeelding 2022-07-03 om 08 12 19" src="https://user-images.githubusercontent.com/108798198/177523935-ec099846-d01b-4df6-ab5b-c801bcb580a6.png">
<img width="1015" alt="Schermafbeelding 2022-07-03 om 08 14 46" src="https://user-images.githubusercontent.com/108798198/177525335-96e9310a-f6db-435b-be73-f1af9c0f8691.png">
<img width="267" alt="Schermafbeelding 2022-07-03 om 08 15 27" src="https://user-images.githubusercontent.com/108798198/177521989-61130d7e-474e-45ab-9402-bbd016a014f4.png">

**Comment by matfish2** [comment]
Hi Paul. Thank you for reporting this. I will have a look later today and update. 

Are the templates the same for all sites, and the difference is just dynamic DB content, or do you need the templates duplicated for all sites?

**Comment by GHPaulB** [comment]
Hi,

thank you for your reply.

I live in Belgium, a small country with three languages, surrounded by four big countries.
As you can imagine, we need complex multilingual / multiregional websites :-)

Each language is a separate site in Craft.
Each site has identical subdirectories.
But each subdirectory has its own template page, that includes a shared basic template (in the _includes folder).
This way I can adjust parts of the website if, for example, I need a different layout for a certain region.

Hope this helps solving the problem.

Kind regards,
Paul Schraepen

**Comment by matfish2** [comment]
As far as I understand there are two issues here.

1. Backend: Sections and Categories are only assigned to Primary site.
This was fixed in 1.0.9 (Just released). Thank you!

Note that this will only take effect on a fresh install.
**Beware that uninstalling will delete data and templates, so if you made changes I'd suggest just adding the sites manually to Sections and Categories.**

2. Frontend: 

https://craftcms.com/docs/3.x/sites.html#step-4-update-your-templates
> If you have any templates that you only want to serve from a specific site, you can create a new subfolder in your templates folder, named after your site’s handle, and place the templates in there.

I think as a starting point it is better not to have the templates duplicated across all sites.
If you have specific parts that you would like to vary between sites you can use something like (see link above):
```
{% if craft.app.language == 'de' %}
  <p>I like bread and beer.</p>
{% endif %}
```

or better yet use polymorphism: (pseudo-code, didn't test)
```
{% include craft.app.language/my-template %} 
```
If you still want to duplicate specific templates, you can copy them to the relevant folder, e.g: 
```
temlpates/nlbe/blogify/listing/_entry
```
In which case you need to go to the relevant section and change the template location for the relevant site.

Please let me know if that makes sense.

**Comment by matfish2** [comment]
Added to [docs](https://github.com/matfish2/craft-blogify#multiple-sites).
> In which case you need to go to the relevant section and change the template location for the relevant site.

Apparently, you don't even have to do this, as long as the subfolder path matches the original path
https://craftcms.com/docs/4.x/sites.html#step-4-update-your-templates
